draft-ietf-acme-acme.md

```diff
@@ -1603,14 +1609,19 @@ Given a Challenge/Response pair, the ACME server verifies the client's control
 of the domain by verifying that the TLS server was configured appropriately,
 using these steps:
 
-1. Compute the Z-value from the key authorization in the same way as the client.
+1. Compute SAN A and SAN B in the same way as the client.
 2. Open a TLS connection to the domain name being validated on the requested
-   port, presenting the value "\<Z[0:32]\>.\<Z[32:64]\>.acme.invalid" in the
-   SNI field (where the comparison is case-insensitive).
-3. Verify that the certificate contains a subjectAltName extension with the
-   dNSName of "\<Z[0:32]\>.\<Z[32:64]\>.acme.invalid", and that no other dNSName
-   entries of the form "*.acme.invalid" are present in the subjectAltName
-   extension.
+   port, presenting SAN B in the SNI field.
+
+   The server MUST request the certificate using an SNI value of SAN B.
```

Comment:
True, i believe either order is sufficient for security, but I think it's desirable (helpful?) to keep with the pattern of `http-01` where the token is used in the request, and the key authorization is provided in the response.